tecken/upload/utils.py

```diff
@@ -25,7 +25,9 @@
 logger = logging.getLogger("tecken")
 metrics = markus.get_metrics("tecken")
 
-downloader = SymbolDownloader(settings.SYMBOL_URLS)
+# FIXME(willkg): This downloader needs to point to all the things that can be downloaded
+# from so it can correctly invalidate the cache
+downloader = SymbolDownloader(settings.SYMBOL_URLS + [settings.UPLOAD_TRY_SYMBOLS_URL])
```

Comment:
This is lousy because we're defining multiple SymbolDownloaders that can get out of sync which was the crux of the problem here.

When we rewrite the storage interface ([1564452](https://bugzilla.mozilla.org/show_bug.cgi?id=1564452)), we will fix this.